tests/components/workday/__init__.py

```diff
@@ -40,6 +40,31 @@ async def init_integration(
     return config_entry
 
 
+TEST_CONFIG_NO_COUNTRY = {
+    "name": DEFAULT_NAME,
+    "excludes": DEFAULT_EXCLUDES,
+    "days_offset": DEFAULT_OFFSET,
+    "workdays": DEFAULT_WORKDAYS,
+    "add_holidays": [],
+    "remove_holidays": [],
+}
+TEST_CONFIG_NO_COUNTRY_ADD_HOLIDAY = {
+    "name": DEFAULT_NAME,
+    "excludes": DEFAULT_EXCLUDES,
+    "days_offset": DEFAULT_OFFSET,
+    "workdays": DEFAULT_WORKDAYS,
+    "add_holidays": ["2020-02-24"],
+    "remove_holidays": [],
+}
+TEST_CONFIG_NO_COUNTRY_WITH_PROVINCE = {
+    "name": DEFAULT_NAME,
+    "excludes": DEFAULT_EXCLUDES,
+    "days_offset": DEFAULT_OFFSET,
+    "workdays": DEFAULT_WORKDAYS,
+    "add_holidays": [],
+    "remove_holidays": [],
+    "province": "NO",
+}
```

Comment:
Missing `country`

Reply:
I'm not 100% sure what you mean, but I think what you're asking for might be covered by `TEST_CONFIG_NO_PROVINCE` below.

Reply:
`country` key should be in these even if it's with a `None` value